crates/goose-mcp/src/developer/mod.rs

```diff
@@ -1559,7 +1561,7 @@ impl DeveloperRouter {
 
                 let window = windows
                     .into_iter()
-                    .find(|w| w.title() == window_title)
+                    .find(|w| w.title().unwrap_or_default() == window_title)
```

Comment:
why would we not have a window title?

Reply:
xcap 0.0.14 -> xcap 0.7.0
```
-    pub fn title(&self) -> &str {
-        &self.impl_window.title
-    }
...
+    pub fn title(&self) -> XCapResult<String> {
+        self.impl_window.title()
+    }
```

There is for the change of xcap.

Or do you think is it good?